.eslintrc.js

```diff
@@ -8,7 +8,6 @@ module.exports = {
     'dist',
     'docs',
     'coverage',
-    '*.d.ts',
```

Comment:
Type definition files are valid code, so I don't think it makes sense to exclude them completely from ESLint.